### PR TITLE
Update how NEXO information is imported

### LINKED ISSUE #2994: Nexo CSV not being correctly imported
## Problem Definition

One user reported that the description field raised a key error. After (s)he provided a CSV mocked file I could verify 
that several columns had different names and there where new columns

## Steps

1. Asked the original author of the issue for more information and if he could confirm that he had the same file
2. If indeed we had outdated information it's only a matter of renaming keys and it should be fixed
3. If there are several valid CSV files we should investigate different approaches 

Data that the user provided

```
Transaction,Type,Currency,Amount,USD Equivalent,Details,Outstanding Loan,Date / Time
NXT0000000001,LockingTermDeposit,EURX,-1,$1.1226435,approved / Transfer from Savings Wallet to Term Wallet,$0.00,2021-03-06 09:51
NXT0000000002,Interest,EURX,1,$1.16806918161,approved / EURx Interest Earned,$0.00,2021-03-06 01:20
NXT0000000003,ExchangeDepositedOn,EUR,10,$15.1411,approved / EUR,$0.00,2021-02-26 19:09
NXT0000000004,DepositToExchange,EUR,100,$120.1509,approved / EUR Top Up,$0.00,2021-02-26 14:58
NXT0000000005,WithdrawExchanged,EUR,-2000.79,$2500.63591346,approved / EUR Withdrawal,$0.00,2020-12-16 15:09
NXT0000000006,ExchangeToWithdraw,EURX,-2000.79,$2500.5364681,approved / User,$0.00,2020-12-16 21:18
NXT0000000007,Dividend,NEXO,10.3585507,$3.43448965222,approved / Loyalty Dividend,$0.00,2020-08-15 12:01
NXT0000000008,ExchangeToWithdraw,EURX,-2000,$2500.5,rejected / USER,$0.00,2020-08-15 01:10
NXT0000000009,Dividend,NEXO,22.5653042,$16.0708424630,approved / Base Dividend,$0.00,2019-08-15 17:01
NXT0000000010,ExchangeDepositedOn,EUR,10000,$11174.47,approved / EURx,$0.00,2019-08-15 23:47
NXT0000000011,Deposit,NEXO,1.00001,$5487.0248909704,approved / 0x999999999849662f1106541b0d2e78be4445e6d8d870ac8616a3cf79de969c9b,$0.00,2019-05-24 17:50
NXT0000000012,Withdrawal,BTC,-0.9995,$5574.2115,approved / BTC withdrawal,$0.00,2019-04-24 16:18
NXT0000000013,Deposit,BTC,1,$5579.7,approved / 3699999999999999999999999,$0.00,2019-04-24 14:42
```

### System Description

Rotki version: 1.17.1

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #2994

Updated the test file. There is one ignored field that I asked the user about and I'm waiting for response

### CODE REVIEW

File: rotkehlchen/data/importer.py [hunk 1]
```diff
@@ -792,10 +792,13 @@ def _consume_nexo(self, csv_row: Dict[str, Any]) -> None:
         - UnknownAsset
         - DeserializationError
         """
-        if 'rejected' not in csv_row['Description']:
+
+        ignored_entries = ('ExchangeToWithdraw', 'DepositToExchange')
+
```
Comment (LefterisJP) [inline rotkehlchen/data/importer.py:797]: unnecessary whitespace

```suggestion
        ignored_entries = ('ExchangeToWithdraw', 'DepositToExchange')
```

File: rotkehlchen/data/importer.py [hunk 2]
```diff
@@ -858,10 +862,12 @@ def _consume_nexo(self, csv_row: Dict[str, Any]) -> None:
                 asset=asset,
                 rate=None,
                 rate_asset=None,
-                link=None,
+                link=transaction,
                 notes=f'{entry_type} from Nexo',
             )
             self.db_ledger.add_ledger_action(action)
+        elif entry_type in ignored_entries:
```
Comment (LefterisJP) [inline rotkehlchen/data/importer.py:869]: I think this approach is not really future-safe. Instead what you should go is ignore any unknown/unexpected entries with a warning.

And iff all the expected entries were not there raise an exception.

**Review by LefterisJP — COMMENTED** [review]
some comments

File: rotkehlchen/data/importer.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/data/importer.py:868]: At the moment I rise a warning if a unknow row is detected. In this case there are two rows that include information for internal operations of nexo and contain information that we get from other rows
